notifications/rest/users/create-user/create-user.7.x.java

```diff
@@ -0,0 +1,27 @@
+// NOTE: This example uses the next generation Twilio helper library - for more
+// information on how to download and install this version, visit
+// https://www.twilio.com/docs/libraries/java
+import com.twilio.Twilio;
+import com.twilio.rest.notify.service.Notification;
+
+public class Example {
+  // Find your Account Sid and Token at twilio.com/user/account
+  public static final String ACCOUNT_SID = "ACCOUNT_SID";
+  public static final String AUTH_TOKEN = "AUTH_TOKEN";
+
+  public static final String SERVICE_SID = "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+  public static void main(String[] args) {
+    // Initialize the client
+    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);
+
+    NotifyUser user = NotifyUser
+        .creator(SERVICE_SID)
+        .create(
+            "User001",
+            new ArrayList<String>("premium", "fitness-lifestyle")
```

Comment:
In other `Creator` classes, the `create` method hasn't got any parameters. Those parameters will probably be set with `setters` with the name of the field being set, something like `setIdentity` and `setSegments`. The `create` method is call afterwards.